File: src/main/java/com/rs/java/game/area/multi/PiscatorisColonyMulti.java
package com.rs.java.game.area.multi;

import com.rs.java.game.area.Area;

public final class PiscatorisColonyMulti extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Piscatoris Colony Multi";
	}

	public PiscatorisColonyMulti() {
		super(9273);
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.MULTI;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/multi/PiscatorisColonyMulti.kt
package com.rs.kotlin.game.world.area.areas.multi

import com.rs.kotlin.game.world.area.Area

class PiscatorisColonyMulti : Area(9273) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Piscatoris Colony Multi"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.MULTI
    }
}
